test/framework/setup.go

```diff
@@ -175,21 +175,21 @@ spec:
 	return YamlK8s(mesh)
 }
 
-func MTLSMeshWithMeshServicesKubernetes(name string, meshServicesEnabled string) InstallFunc {
```

Comment:
Can we remove them to use `samples.MeshBuilder()...` in the test?

Reply:
It's completely unrelated to this and shouldn't be in this PR

Reply:
we have https://github.com/kumahq/kuma/issues/11294

Reply:
done